x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/types.ts

```diff
@@ -7,17 +7,30 @@
 
 export type BoostType = 'value' | 'functional' | 'proximity';
 
-export interface Boost {
-  type: BoostType;
+export interface BaseBoost {
   operation?: string;
   function?: string;
+}
+
+export interface Boost extends BaseBoost {
+  type: BoostType;
   newBoost?: boolean;
   center?: string | number;
-  value?: string | number | string[] | number[];
+  value?: Array<string | number>;
   factor: number;
 }
 
+export interface BoostObject {
+  [key: string]: Boost[];
+}
+
```

Comment:
I thought we replaced this with `Record` in a previous PR - did this get added back in by mistake?

```suggestion
```

Reply:
😱 Let me double check that

Reply:
So we *did* move to `Record` in `SearchSettings.boosts`, but it looks like I likely forgot to kill this type (which is no longer used anywhere).

Reply:
Removed in f47749c1eee7ac6ecf1016cd91e905fdd12f8fce